automated/linux/ltp-open-posix/ltp-open-posix.sh

```diff
@@ -63,6 +66,16 @@ parse_ltp_output() {
 
 }
 
+# Parse LTP open posix grp output
+parse_ltp_grp_output() {
+    sed -i '/^PASS\|^FAIL\|^Testing/!d' ${GRP_OUTPUT_FILE}
```

Comment:
LTP open posix test suites results verdicts / Labels are different than pass, fail and skip.
Please consider below labels also.
SKIPPED|: UNSUPPORTED|: UNTESTED|: UNRESOLVED|: HUNG

Reply:
Okay. Let me add these labels too.

Reply:
Handled these labels and tested the changes with this job: https://lava.collabora.co.uk/scheduler/job/4156348